pkg/collector/python/containers.go

```diff
@@ -41,7 +41,7 @@ func IsContainerExcluded(name, image, namespace *C.char) C.int {
 		goNs = C.GoString(namespace)
 	}
 
-	if filter.IsExcluded(goName, goImg, goNs) {
+	if filter.IsExcluded(nil, goName, goImg, goNs) {
```

Comment:
This code is only used by the `Kubelet` check. Should we also add filtering by pod annotations for Python code ?